ledger-core/virtual/object/object.go

```diff
@@ -428,8 +435,63 @@ func (sm *SMObject) stepMigrate(ctx smachine.MigrationContext) smachine.StateUpd
 		return ctx.Error(fmt.Errorf("failed to publish state report"))
 	}
 
+	sdlCallSummarySync := ctx.Share(&sm.SummaryDone, 0)
```

Comment:
Can that new code be extracted from here into callsummary package?

Reply:
Sure, we can, for what?